### ISSUE TITLE
keras2onnx success,but the accuracy become too low to use

### ISSUE THREAD

**Issue by KingerFzb** [onnx/onnx#2685]
`import keras
import keras2onnx
import onnx
from ssd_layers import Normalize,PriorBox
from keras.models import load_model
model = load_model('vgg_ssd.h5',custom_objects={'Normalize': Normalize,'PriorBox':PriorBox})  
onnx_model = keras2onnx.convert_keras(model, model.name)
temp_model_file = 'model/model_ssd_vgg.onnx'
onnx.save_model(onnx_model, temp_model_file)
print('change successful')
`
the vgg_ssd accuracy 90% before,and become 10% after change

**Comment by linkerzhang** [comment]
@wenbingl @EmmaNingMS

**Comment by wenbingl** [comment]
@KingerFzb can you share more information about this vgg_ssd model? Or compare the raw result of onnx model output to the original model and share more details?

**Comment by KingerFzb** [comment]
I have create some layers that keras aren`t have,like Normalize,PriorBox,i use custom_objects to do that，i don't know if keras2onnx can support the layers design by myself

> @KingerFzb can you share more information about this vgg_ssd model? Or compare the raw result of onnx model output to the original model and share more details?

**Comment by wenbingl** [comment]
> I have create some layers that keras aren`t have,like Normalize,PriorBox,i use custom_objects to do that，i don't know if keras2onnx can support the layers design by myself
> 
> > @KingerFzb can you share more information about this vgg_ssd model? Or compare the raw result of onnx model output to the original model and share more details?

Yes,  the converter can convert the custom layer itself by parsing the corresponding tensorflow ops used in the custom layer.

**Comment by faxu** [comment]
@KingerFzb were you able to resolve this or do you still require support for investigation?

**Comment by askhade** [comment]
Closing this as not sure if this is still an issue...
